tests/lib/lte_lc_api/src/lte_lc_api_test.c

```diff
@@ -254,7 +259,6 @@ void test_lte_lc_deregister_handler_null(void)
 	TEST_ASSERT_EQUAL(-EINVAL, ret);
 }
 
-
```

Comment:
There is another one of this extra newlines right under `sys_init_helper`.

Reply:
This is a removal of an extra empty line to make 2 empty lines between functions to 1 empty line. So I guess this is fine, right?
I've removed an extra empty line after `sys_init_helper`.